res/translations/es.json

```diff
@@ -0,0 +1,230 @@
+    "album": "álbum",
+    "Album count": "Número de álbumes",
+    "Album Count": "Número de álbumes",
+    "Album filters": "Filtros de álbumes",
+    "Album gain": "Ganancia del álbum",
+    "Album info not available": "Información del álbum no disponible",
+    "Album peak": "Pico del álbum",
+    "Albums": "Álbumes",
+    "albums": "álbumes",
+    "All": "Todo",
+    "All Tracks": "Todas las pistas",
+    "Alt. URL": "URL alternativa",
+    "Are you sure you want to delete the server": "¿Estás seguro de que quieres eliminar el servidor?",
+    "Artist": "Artista",
+    "Artist (A-Z)": "Artista (A-Z)",
+    "Artists": "Artistas",
+    "Artist biography not available.": "Biografía del artista no disponible.",
+    "Audio device": "Dispositivo de audio",
+    "Audio Drama": "Drama de audio",
+    "Audiobook": "Audiolibro",
+    "Authentication failed": "Autenticación fallida",
+    "Auto": "Automático",
+    "Autoselect device": "Seleccionar dispositivo automáticamente",
+    "Back": "Atrás",
+    "Bit rate": "Tasa de bits",
+    "BPM": "BPM",
+    "Broadcast": "Transmisión",
+    "Cancel": "Cancelar",
+    "Check for Updates": "Buscar actualizaciones",
+    "Close": "Cerrar",
+    "Close to system tray": "Cerrar a la bandeja del sistema",
+    "Comment": "Comentario",
+    "Compilation": "Compilación",
+    "Composer": "Compositor",
+    "Configure your music server to add radio stations": "Configura tu servidor de música para agregar estaciones de radio",
+    "Confirm Delete Playlist": "Confirmar eliminación de la lista de reproducción",
+    "Confirm Delete Server": "Confirmar eliminación del servidor",
+    "Connect to Server": "Conectar al servidor",
+    "Connecting": "Conectando",
+    "Connecting to": "Conectando a",
+    "Content type": "Tipo de contenido",
+    "Could not reach server": "No se pudo conectar con el servidor",
+    "Create new playlist": "Crear nueva lista de reproducción",
+    "day": "día",
+    "days": "días",
+    "Delete Playlist": "Eliminar lista de reproducción",
+    "Demo": "Demo",
+    "Description": "Descripción",
+    "Disable server transcoding": "Desactivar transcodificación del servidor",
+    "Disc number": "Número de disco",
+    "Discography": "Discografía",
+    "discs": "discos",
+    "DJ-Mix": "Mezcla de DJ",
+    "Download": "Descargar",
+    "Download completed": "Descarga completa",
+    "Duration": "Duración",
+    "Edit": "Editar",
+    "Edit Playlist": "Editar lista de reproducción",
+    "Edit server": "Editar servidor",
+    "Enable system tray": "Habilitar bandeja del sistema",
+    "Enabled": "Habilitado",
+    "Enter": "Entrar",
+    "EP": "EP",
+    "Equalizer": "Ecualizador",
+    "Exclusive mode": "Modo exclusivo",
+    "Favorites": "Favoritos",
+    "Field Recording": "Grabación en campo",
+    "File path": "Ruta al archivo",
+    "File size": "Tamaño del archivo",
+    "Filter albums": "Filtrar álbumes",
+    "Filter genres": "Filtrar géneros",
+    "Forward": "Adelantar",
```

Comment:
Is this in reference to forward in the track (As in fast forward) or navigation (as opposed to back).

Reply:
Navigation through pages, forward and back